apps/core/app/[locale]/(default)/account/[tab]/page.tsx

```diff
@@ -37,10 +40,10 @@ const tabHeading = async (heading: string, locale: string) => {
 export default async function AccountTabPage({ params: { tab, locale }, searchParams }: Props) {
   switch (tab) {
     case 'orders':
-      return tabHeading(tab, locale);
+      return <TabHeading heading={tab} locale={locale} />;
```

Comment:
Do we need to pass locale here? Couldn't get use `getLocale` inside the component instead?

Reply:
@jorgemoya great idea! Refactored all over the accont